CONTRIBUTING.md

```diff
@@ -10,19 +10,31 @@ expectations the Node.js Foundation requires from all contributors.
 
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a contributor who has been given write access to the repository
-* An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+* An **Observer** is any individual who has requested or been requested to attend a CommComm meeting.
 * A **Member** is a collaborator with voting rights who has met the requirements of participation to be considered for acceptance, and subsequently voted in by the CommComm voting process.
```

Comment:
Since we don't know yet which requirements there will have, maybe just "...who has met the requirements to be considered..."?

Reply:
This is the language proposed for GOVERNANCE.md:

`Active Collaborators of a CC initiative or working group can be nominated by
a CommComm Member to become a Member of CommComm.`

My interpretation is that `participation`, though vague, describes whatever we may define in GOVERNANCE as the expectations from a Member. Of course, we are looking to improve the language where we can and suggestions are welcome! My understanding is GOVERNANCE contains the requirements, and other docs (README, CONTRIBUTION etc.) defer to it for the formal definitions.